index.js

```diff
@@ -26,23 +26,35 @@ class PersonalDataFilter {
 	}
 
 	filter(data) {
+		const result = this._filterRecursively(data, []);
```

Comment:
Can't we use map and get items by index instead of using find?

Reply:
Not really:
```Nodejs
$ node
> map = {}
{}
> testObj1 = { a: 5}
{ a: 5 }
> testObj2 = { b: 4}
{ b: 4 }
> map[testObj1] = true
true
> map
{ '[object Object]': true }
> map[testObj2] === true
true
```